Source/Core/Core/HW/SI/SI_DeviceGBA.cpp

```diff
@@ -300,8 +300,8 @@ int CSIDevice_GBA::RunBuffer(u8* buffer, int request_length)
     if (m_sock_server.Connect())
     {
 #ifdef _DEBUG
-      NOTICE_LOG(SERIALINTERFACE, "%01d cmd %02x [> %02x%02x%02x%02x]", m_device_number, buffer[0],
-                 buffer[1], buffer[2], buffer[3], buffer[4]);
+      NOTICE_LOG_FMT(SERIALINTERFACE, "{:01d} cmd {:02x} [> {:02x}{:02x}{:02x}{:02x}]",
```

Comment:
I realise this is just a direct translation of the original format string, but could `{:01d}` not just be `{}`? `d` is redundant with {fmt}, and so is the `01` since there will always be at least one digit...